bin/setup_integration_test.sh

```diff
@@ -6,18 +6,10 @@
 
 # Sets up S3 buckets for integration test.
 
-if [ -z "${SUBMITTER_S3_BUCKET}" ]; then
-    echo "SUBMITTER_S3_BUCKET is not set."
-    exit 1;
-fi
+set -euo pipefail
 
-if [ -z "${CRASHSTORAGE_BUCKET_NAME}" ]; then
-    echo "CRASHSTORAGE_BUCKET_NAME is not set."
-    exit 1;
-fi
-
-./bin/aws_s3.sh rb --force s3://${SUBMITTER_S3_BUCKET}/ 2>&1 > /dev/null
+./bin/aws_s3.sh rb --force s3://${SUBMITTER_S3_BUCKET}/ || true
 ./bin/aws_s3.sh mb s3://${SUBMITTER_S3_BUCKET}/
 
-./bin/aws_s3.sh rb --force s3://${CRASHSTORAGE_BUCKET_NAME}/ 2>&1 > /dev/null
+./bin/aws_s3.sh rb --force s3://${CRASHSTORAGE_BUCKET_NAME}/ || true
```

Comment:
This was printing stuff out anyhow, so I nixed all the out/err redirection which apparently wasn't working. Further, because this script now dies if there's a non-zero exit code, we add `|| true`.